UI/src/app/services/store.service.ts

```diff
@@ -12,7 +12,10 @@ export class StoreService {
     target: [],
     targetConfig: {},
     source: [],
-    report: undefined
+    report: undefined,
+    sourceColumnsfilter: undefined,
```

Comment:
We should name it `search` to not confuse with target tables filter
Let's use one key `search` inside of state with keys of areas instead of three keys

```ts
search: {
  source: undefined,
  target: undefined,
  sourceColumns: undefined
}

```

Reply:
Done